elliott/elliottlib/cli/find_builds_cli.py

```
@@ -723,23 +748,21 @@ async def find_builds_konflux(runtime, payload):
         image_metas.append(image)
 
     LOGGER.info("Fetching NVRs from DB...")
-    tasks = [
-        image.get_latest_build(el_target=image.branch_el_target(), exclude_large_columns=True) for image in image_metas
-    ]
-    records: List[Dict] = [r for r in await asyncio.gather(*tasks) if r is not None]
+    tasks = [_get_latest_build_by_network_mode_config(image) for image in image_metas]
+    records: list[dict] = [r for r in await asyncio.gather(*tasks) if r is not None]
     if len(records) != len(image_metas):
         raise ElliottFatalError(f"Failed to find Konflux builds for {len(image_metas) - len(records)} images")
```

Ruff (0.14.11)

Avoid specifying long messages outside the exception class

(TRY003)